lib/cosmos/gui/qt.rb

```diff
@@ -175,6 +175,24 @@ def self.getFont(font_face, font_size, font_attrs = nil, font_italics = false)
     font
   end
 
+  # Get the default small font for the platform (Windows, Mac, Linux)
+  def self.get_default_small_font
```

Comment:
I'm not completely sold on the method name but we had this logic spread out all over the place.